utils/params_test.go

```diff
@@ -279,10 +275,9 @@ func extractAndAssertParamsFromEnv(t *testing.T, platformUrl, basicAuth bool) {
 		assert.Equal(t, "frogbot", configParams.RepoName)
 		assert.Equal(t, "123456789", configParams.Token)
 		// ScanRepository command context
-		if len(configParams.Branches) != 0 {
+		if commandName == ScanRepository {
 			assert.Equal(t, "dev", configParams.Branches[0])
 			assert.Equal(t, int64(0), configParams.PullRequestDetails.ID)
-			assert.Equal(t, "my-custom-commit-template", configParams.Git.CommitMessageTemplate)
```

Comment:
removed template check as this is tested in other function and this is not generic to the test